### DISCUSSION TITLE
Add TTL to runs stored in conversational memory

### DISCUSSION THREAD

**Discussion by igor-2lemetry** [griptape-ai/griptape#1527] [Ideas]
- [x] I have read and agree to the [contributing guidelines](https://github.com/griptape-ai/griptape#contributing).

**Is your feature request related to a problem? Please describe.**
It would be great to exclude from prompt stack too old runs.

**Describe the solution you'd like**
Possible it should be done by adding ttl attribute in ConversationMemory and some timestamp for each run.

**Describe alternatives you've considered**
Alternatives are: pruning by max_runs or tokens_left.

**Additional context**
No additional context.

**Comment by collindutter** [comment]
@igor-2lemetry I think this is best suited for the database to handle, what do you think?
